packages/web-app-files/src/components/AppBar/SelectedResources/SizeInfo.vue

```diff
@@ -11,7 +11,7 @@
     </translate>
     <translate
       v-else
-      key="multiple-select-info-with-size"
+      key="multiple-select-info"
```

Comment:
this section only shows number of selected so I think the key should be `multiple-select-info`